raiden-ts/src/helpers.ts

```diff
@@ -507,7 +503,7 @@ export async function getState(
     log,
   }: Pick<RaidenEpicDeps, 'address' | 'network' | 'contractsInfo' | 'log'>,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  storage: { state?: any; storage?: Storage } & RaidenDatabaseOptions = {},
+  storage: { state?: any; } & RaidenDatabaseOptions = {},
```

Comment:
This was intended to allow users to pass further PouchDB options to this object, but since this isn't exposed in `Raiden.create`'s interface, the sole user of this helper function, we should simplify it.
```suggestion
  storage: { state?: any; adapter?: any; prefix?: string } = {},
```

Reply:
:heavy_check_mark: